File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_mall_home_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusMallHomeVO:
    """Mall Home Value Object - Represents the home page data of the mall including various product lists"""
    created_at: str = None
    updated_at: str = None
    grid_categories: List[PlusCategoryVO] = None
    products: List[PlusProductVO] = None
    banners: List[PlusProductVO] = None
    hot_products: List[PlusProductVO] = None
    new_products: List[PlusProductVO] = None
    recommend_products: List[PlusProductVO] = None
    hot_sales_products: List[PlusProductVO] = None

File: sdkwork-backend-sdk-go/types/plus_mall_home_vo.go
package types

// Mall Home Value Object - Represents the home page data of the mall including various product lists
type PlusMallHomeVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	GridCategories []PlusCategoryVO `json:"gridCategories"`
	Products []PlusProductVO `json:"products"`
	Banners []PlusProductVO `json:"banners"`
	HotProducts []PlusProductVO `json:"hotProducts"`
	NewProducts []PlusProductVO `json:"newProducts"`
	RecommendProducts []PlusProductVO `json:"recommendProducts"`
	HotSalesProducts []PlusProductVO `json:"hotSalesProducts"`
}
